### ISSUE TITLE
Hpatches evaluataion result

### ISSUE THREAD

**Issue by panxkun** [PruneTruong/GLU-Net#5]
Hi, Thanks for the great work! I have a question about the Hpatches evaluation on the aepe and pck metric. I use the script `eval.py` in this repo :
` python eval.py --pre_trained_models DPED_CityScape_ADE --data_dir path_to_hpatches --hpatches_original_size True`
Then it generated a result file named ./evaluation/metrics_GLUNet_HpatchesDataset.txt. I can read the mean value of the aepe( I added it in your code) , pck-1 and pck-5 metric in every scene for 1 to 5, so I dircetly compute the total aepe and pck-1 pck-5, but there is a big difference between the result I got and that in your paper.

|         |AEPE|PCK-1| PCK-5|AEPE|PCK-1| PCK-5|
|  ----  | ----  | ----  | ----  | ----  |----  | ----  |
| in paper | 7.40 | 59.92%|83.47%| 25.05 | 39.55%|78.54|
| code | 8.04|19,18%|83.11%| 25.85| 13.20%|75.09%|

from col 1 to col 3 is HP 240x240, from col 4 to col 6 is HP. There is a clear gap in pck-1 but the rest metric including aepe and pck-5 is close. So I was doubt whether I run the code in wrong way.

**Comment by PruneTruong** [comment]
Hi, I just ran the code from the repo, this is what I obtain on average for the 5 scenes on HP-240: 
{"final_eape":7.399756344294144,"pck_alpha_0_05_average_per_image":0.8929365867510388,"pck_thresh_1_average_per_image":0.5991927236584812,"pck_thresh_5_average_per_image":0.8346908606941198}. 

Which torch version are you using ? I am using torch1.0 with cuda 9.0 (but cuda10.0 shouldn't make a difference)

**Comment by panxkun** [comment]
thanks, the version of pytorch indeed influence the result of the evaluation, after I change the version of pytroch from 1.6 to 1.0.0, I got the same result of that in paper

**Comment by PruneTruong** [comment]
Hi, just so you know, I now adapted the code for it to run on torch 1.6/1.7 (the default behavior of grid_sample changed) so there shouldn't be any issues anymore. However, i havent tried training on higher torch version yet, so i cannot guarantee smooth training there at the moment !
